db/commitlog/commitlog.cc

```diff
@@ -323,6 +334,8 @@ class db::commitlog::segment_manager : public ::enable_shared_from_this<segment_
     requires std::derived_from<T, db::commitlog::entry_writer> && std::same_as<R, decltype(std::declval<T>().result())>
     future<R> allocate_when_possible(T writer, db::timeout_clock::time_point timeout);
 
+    future<> oversized_allocation(entry_writer&, db::timeout_clock::time_point, size_t);
+
```

Comment:
Please add names to the parameters. It's so much easier to understand.

Reply:
Done